### PR TITLE
Spellcheck @doc/examples/segmentation/plot_watershed.py

### PR DESCRIPTION
**Pull request by BBQuercus**
## Checklist

- [x] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [x] Gallery example in `./doc/examples` (new features only)
- [x] Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- [x] Unit tests
- [x] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)

## For reviewers

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.
- [ ] Consider backporting the PR with `@meeseeksdev backport to v0.14.x`

### CODE REVIEW

File: doc/examples/segmentation/plot_watershed.py [hunk 1]
```diff
@@ -8,15 +8,15 @@
...
 meet on watershed lines.  In many cases, markers are chosen as local
 minima of the image, from which basins are flooded.
 
 In the example below, two overlapping circles are to be separated. To
 do so, one computes an image that is the distance to the
 background. The maxima of this distance (i.e., the minima of the
-opposite of the distance) are chosen as markers, and the flooding of
-basins from such markers separates the two circles along a watershed
+opposite of the distance) are chosen as markers and the flooding of
+basins from such markers separate the two circles along a watershed
```
Comment (soupault) [inline doc/examples/segmentation/plot_watershed.py]: This doesn't look correct

Comment (BBQuercus) [reply doc/examples/segmentation/plot_watershed.py]: Which one are you referring to?

Comment (hmaarrfk) [reply doc/examples/segmentation/plot_watershed.py]: I think it still needs the `s`

Comment (BBQuercus) [reply doc/examples/segmentation/plot_watershed.py]: True, I agree. Put the `s` back.

**Comment by hmaarrfk** [comment]
Thanks!

**Comment by BBQuercus** [comment]
👍 Thank you too.

**Pull request merged**